sourcetools/com.ibm.uma/com/ibm/uma/om/Module.java

```diff
@@ -64,7 +64,13 @@ public void addArtifact(Artifact artifact) throws UMAException {
 	}
 	
 	public void addExports(String group, Exports exps) {
-		exports.put(group, exps);
+		if ( exports.containsKey(group) ) {
+			Exports allGroupExports = exports.get(group);
+			allGroupExports.addExports(exps);
+			exports.replace(group, allGroupExports);
```

Comment:
This shouldn't be needed, since the Exports in the Map was already modified on the previous line.

Reply:
Right, corrected, thx!